### PR TITLE
Replace Leave-project-empty button with Upload-project when file is provided

### LINKED ISSUE #538: Reset project shows 'Leave project empty' button during upload
During the upload of a zip file for a reset the `Leave project empty` is left enabled and is very tempting to click while you wait for the upload. We should instead disable that button since they probably don't want to abort during an upload.

### PR DESCRIPTION
**Pull request by myieye**
Resolves #538 

https://github.com/sillsdev/languageforge-lexbox/assets/12587509/2bcd0fbd-88fa-4115-977f-a0d00cf664f7

### CODE REVIEW

File: frontend/src/lib/components/TusUpload.svelte [hunk 1]
```diff
@@ -1,43 +1,50 @@
...
+  import { createEventDispatcher, onDestroy, onMount } from 'svelte';
+  import t from '$lib/i18n';
+  import IconButton from './IconButton.svelte';
 
   export let endpoint: string;
   export let accept: string;
-  export let uploadLabel: string = $t('tus.upload');
   export let inputLabel: string = $t('tus.select_file');
   export let inputDescription: string | undefined = undefined;
+  export let internalButton = false;
```
Comment (myieye) [inline frontend/src/lib/components/TusUpload.svelte:24]: I don't really like how I made the button external, but...I also don't want to spend lots of time on this.
If we need more file-inputs, then we can revisit it. I think this is acceptable for now.

File: frontend/src/routes/(authenticated)/project/[project_code]/SendReceiveUrlField.svelte [hunk 2]
```diff
@@ -31,7 +31,7 @@
       <input value={projectHgUrl} class="input input-bordered join-item w-full focus:input-success" readonly />
       <div class="join-item tooltip-open" class:tooltip={copiedToClipboard} data-tip={$t('clipboard.copied')}>
         {#if copiedToClipboard}
-          <IconButton disabled icon="i-mdi-check" style="btn-outline btn-success" />
+          <IconButton fake icon="i-mdi-check" style="btn-outline btn-success" />
```
Comment (hahn-kev) [inline frontend/src/routes/(authenticated)/project/[project_code]/SendReceiveUrlField.svelte:34]: does fake just mean this is just an icon?

**Review by hahn-kev — APPROVED** [review]
looks good to me

File: frontend/src/routes/(authenticated)/project/[project_code]/SendReceiveUrlField.svelte (hunk 2 above)
Comment (myieye) [reply frontend/src/routes/(authenticated)/project/[project_code]/SendReceiveUrlField.svelte:34]: Yeah 😕. We just happen to have this scenario where it's desireable to replace a real button with a fake button that looks the same, doesn't have ugly "disabled styling", but doesn't actually do anything.

**Pull request merged**